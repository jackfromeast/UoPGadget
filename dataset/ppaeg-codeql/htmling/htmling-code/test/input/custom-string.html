<div><raw-html content="here <br /> my friend"/></div>
