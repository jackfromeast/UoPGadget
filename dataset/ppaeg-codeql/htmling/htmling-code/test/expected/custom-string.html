<div>here <br /> my friend</div>
